pkg/monitor/monitor/util.go

```diff
@@ -423,17 +424,14 @@ func getMonitorDMInitContainer(monitor *v1alpha1.TidbMonitor, dc *v1alpha1.DMClu
 }
 
 func getMonitorPrometheusContainer(monitor *v1alpha1.TidbMonitor, tc *v1alpha1.TidbCluster, dc *v1alpha1.DMCluster) core.Container {
+	commands := []string{"sed 's/$NAMESPACE/'\"$NAMESPACE\"'/g;s/$POD_NAME/'\"$POD_NAME\"'/g' /etc/prometheus/prometheus.yml > /tmp/prometheus.yml && /bin/prometheus --web.enable-admin-api --web.enable-lifecycle --config.file=/tmp/prometheus.yml --storage.tsdb.path=/data/prometheus " + fmt.Sprintf("--storage.tsdb.retention=%dd", monitor.Spec.Prometheus.ReserveDays)}
```

Comment:
How can we implement the feature that we can update the configuration of Prometheus dynamically if we put the configuration file to /tmp dir in the Prometheus container?

Reply:
After discussion, we put new generated configuration file  to `/etc/prometheus/config_out` folder and mount original configmap to new folder `/etc/prometheus/config`.